### PR TITLE
Add documentation and set log path

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@jbeezley @aashish24 Please review.

File: cdatweb/apps/vtk_view/views.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 try:
     from cdatweb.settings.local_settings import base_path
 except ImportError:
-    base_path = '/data'
+    base_path = 'tmp/data'
```
Comment (aashish24) [inline cdatweb/apps/vtk_view/views.py]: why tmp?

Comment (danlipsa) [inline cdatweb/apps/vtk_view/views.py]: This is saved in the src directory.

**Comment by danlipsa** [comment]
@aashish24 @jbeezley ping?

**Comment by aashish24** [comment]
LGTM :+1:

**Pull request merged**